test/e2e.js

```diff
@@ -52,7 +60,8 @@ const main = async (database, args) => {
     await cleanTestApp(appName);
     await generateTestApp({ appName, database });
 
-    await runAllTests(args).catch(() => {
+    await runAllTests(args).catch(e => {
+      console.error(e);
```

Comment:
you can remove this I guess

Reply:
In some cases the error message was not printed in the shell without it. I think it's better to keep it then. Wdyt?

Reply:
Is it logging twice in some cases then?

Reply:
It's not logged twice but we may want to avoid certain error logs, I'll change the code

Reply:
updated 👌